packages/fxa-graphql-api/src/decorators.ts

```diff
@@ -42,6 +42,9 @@ export const GqlXHeaders = createParamDecorator(
   (data: unknown, context: ExecutionContext): Headers => {
     const ctx = GqlExecutionContext.create(context).getContext();
     const headers = ctx.req?.headers;
+    // We don't need to propagate the authorization token because
+    // it needs to be updated to originate from gql server.
+    delete headers['authorization'];
```

Comment:
In the existing Settings app, aren't we sending a few emails that include location? What's different now?

Reply:
In the settings app, we used the session token to interact with gql api, so that wouldn't have popped up. The `authorization` header is set when using password forgot tokens and account reset tokens.